src/partial-result-stream.js

```diff
@@ -155,41 +134,4 @@ function partialResultStream(requestFn, options) {
     .pipe(userStream);
 }
 
-/**
- * Format a PartialResultSet response from the API. A row object will be created
- * to map each field name to its decoded value.
- *
- * If multiple rows exist in a single PartialResultSet, an array is returned.
- *
- * @param {object} row - A `PartialResultSet` object.
- */
-partialResultStream.formatRow_ = function(metadata, row) {
-  var fields = metadata.rowType.fields;
-
-  if (row.values.length > fields.length) {
-    // More than one row exists. Return an array of formatted rows.
-    var valueSets = chunk(row.values, fields.length);
-
-    return valueSets.map(function(valueSet) {
-      row.values = valueSet;
-      return partialResultStream.formatRow_(metadata, row);
-    });
-  }
-
-  var formattedRow = row.values.map(function(value, index) {
-    var field = fields[index];
-    return {
-      name: field.name,
-      value: codec.decode(value, field),
```

Comment:
Removing this step led to a system test meltdown. I've re-worked the way we format/decode values in the latest commit. One test is still failing.

Reply:
the actual code fails also. there are odd values comiing through.
```
code: 'ERR_ASSERTION',
  actual: 'undefinedunt_created_on_     18215',
  expected: 'account_created_on_     18215',
  operator: '==' }
```

Reply:
Yikes. Can you send over the test file?

Reply:
https://gist.github.com/crwilcox/fa8bb2fb164afdc29747c489ee34ad8a